src/rrules/blas.jl

```diff
@@ -1181,7 +1196,9 @@ function generate_hand_written_rrule!!_test_cases(rng_ctor, ::Val{:blas})
         #
 
         # gemv!
-        map_prod(t_flags, [1, 3], [1, 2], Ps, αs, βs) do (tA, M, N, P, α, β)
+        map_prod(t_flags, [1, 3], [1, 2], allPs, [αs..., 0.46+0.32im], [βs..., 0.39+0.27im]) do (tA, M, N, P, α, β)
```

Comment:
```suggestion
        map_prod(
            t_flags,
            [1, 3],
            [1, 2],
            allPs,
            [αs..., 0.46 + 0.32im],
            [βs..., 0.39 + 0.27im],
        ) do (tA, M, N, P, α, β)
```
Formatting (line length). I'm quite surprised that the formatter didn't comment on this.